packages/components/icons/README.md

```diff
@@ -105,3 +105,34 @@ const app = () => <LeadingIcon icon={<ExportIcon />} />;
 ### Where to use
 
 This component can be used wherever it is necessary to display a themed icon.
+
+## Custom Icon
+
+The custom icon is a an eye-catching visual element that should be used when an additional visual prominence is needed for displaying a custom SVG icon in the UI.
+
+The component is exported as a separate entry point:
+
+```js
+import CustomIcon from '@commercetools-uikit/icons/custom-icon';
+```
+
+### Usage
+
+```js
+import CustomIcon from '@commercetools-uikit/icons/custom-icon';
+import { YourCustomIcon } from './your-custom-icon-directory';
+
+const app = () => <Icon icon={<YourCustomIcon />} />;
+```
+
+### Properties
+
+| Props       | Type                                                      | Required | Values                                              | Default | Description                                                                                                     |
+| ----------- | --------------------------------------------------------- | :------: | --------------------------------------------------- | ------- | --------------------------------------------------------------------------------------------------------------- |
+| `size`      | `string`                                                  |          | '10', '20', '30', '40'                              | '20'    | Specifies the icon size                                                                                         |
+| `icon`      | `union`<br/>Possible values:<br/>`, ReactElement, string` |    -     | A `ReactNode` or `string` that display a custom SVG |         | Icon that is displayed within the component, you must supply a child icon with with this prop or the `svg` prop |
```

Comment:
Is seems the description is out of sync.
I think we need to generate the readme again.